pages/api/propose-post.ts

```diff
@@ -0,0 +1,69 @@
+import { Wallet } from 'ethers';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import { getNowTimestamp } from 'common-util/functions/time';
+import type { ContributeModuleDetails } from 'types/moduleDetails';
+
+// TODO: Update this once we point to prod.
+const AGENT_TYPE = 1;
+const ATTRIBUTE_TYPE_ID = 4;
+
+const ENDPOINT_URL = '/api/agent-attributes';
+const BASE_URL = `${process.env.NEXT_PUBLIC_AFMDB_URL}${ENDPOINT_URL}`;
+const ERROR_MESSAGE = 'Failed to post tweet';
+
+const agentAttr = {
+  agent_id: AGENT_TYPE,
+  attr_def_id: ATTRIBUTE_TYPE_ID,
+  integer_value: null,
+  float_value: null,
+  string_value: null,
+  boolean_value: true,
+  date_value: null,
+  last_updated: new Date().toISOString(),
+};
```

Comment:
there's a chance the could be changed after we open the app and before make an update, that's why I first call GET on the same endpoint. also I think "last_updated" is set by BE